include/robot_localization/ros_filter.h

```diff
@@ -218,6 +250,18 @@ template<class T> class RosFilter
     void imuCallback(const sensor_msgs::Imu::ConstPtr &msg, const std::string &topicName,
       const CallbackData &poseCallbackData, const CallbackData &twistCallbackData,
       const CallbackData &accelCallbackData);
+    
+    //! @brief Callback method for receiving all IMU dynamic correction data
+    //! @param[in] msg - The ROS IMU message to take in.
+    //! @param[in] topicName - The topic name for the IMU message (only used for debug output)
+    //! @param[in] poseCallbackData - Relevant static callback data for orientation variables
+    //! @param[in] twistCallbackData - Relevant static callback data for angular velocity variables
+    //! @param[in] accelCallbackData - Relevant static callback data for linear acceleration variables
+    //!
+    //! This method separates out the orientation, angular velocity, and linear acceleration data and
+    //! passed each on to its respective callback.
+    //!
+    void imuDynamicCorrectionCallback(const nav_msgs::Odometry::ConstPtr &msg, const std::string &topicName);
```

Comment:
You list 5 input params in documentation above method, but the callback only takes in msg and topicName, FYI.

Reply:
Thank you. The whole doc was off. I fixed the parameters and the information.